notebooks/220-license-plate-recognition/216-license-plate-recognition.ipynb

```diff
@@ -0,0 +1,405 @@
+    "First, we initialize OpenVINO Runtime. Then we read the network architecture and model weights from the .bin and .xml files to compile for the desired device. You can choose manually CPU, GPU, MYRIAD etc.\n",
+    "\n",
+    "If you want OpenVINO to decide which hardware offers the best performance, you need to use `AUTO`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# output path for the conversion\n",
+    "converted_model_path = f\"model/intel/{model_name}/{precision}/{model_name}.xml\"\n",
+    "\n",
+    "# initialize inference engine\n",
+    "ie_core = Core()\n",
+    "# read the network and corresponding weights from file\n",
+    "model = ie_core.read_model(model=converted_model_path)\n",
+    "# compile the model for the CPU (you can choose manually CPU, GPU, MYRIAD etc.)\n",
+    "# or let the engine choose the best available device (AUTO)\n",
+    "compiled_model = ie_core.compile_model(model=model, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes\n",
+    "input_layer = compiled_model.input(0)\n",
+    "output_layer = compiled_model.output(0)\n",
+    "\n",
+    "# get input size\n",
+    "input_height, input_width = list(input_layer.shape)[2:4]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    },
+    "tags": []
+   },
+   "source": [
+    "## Inference\n",
+    "\n",
+    "### Using ImageLoader to Process Image\n",
+    "\n",
+    "\n",
+    "> _Model Limitations: Only \"blue\" license plates, which are common in public, were tested thoroughly. Other types of license plates may underperform._\n",
+    "\n",
+    "\n",
+    "#### Model Input:\n",
+    "\n",
+    "name: \"data\" , shape: [1x3x24x94] - An input image in following format [1xCxHxW]. Expected color order is BGR.\n",
+    "\n",
+    "name: \"seq_ind\" , shape: [88, 1] - An auxiliary blob that is needed for correct decoding. Set this to [1, 1, ..., 1].\n",
+    "\n",
+    "#### Notes:\n",
+    "Since the license plate image could be any size, we need to use ImageLoader to resize it to fit the model input requirements.\n",
+    "Augmentation method on image is optional and it may lead to the wrong recognition result. Thus, we recommend you to use it under special conditions like image overexposure or too dark."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def show_image(\n",
+    "    origin_image: np.ndarray, input_image: np.ndarray\n",
+    ") -> matplotlib.figure.Figure:\n",
+    "    \"\"\"\n",
+    "    Visualise how the image we processed.\n",
+    "\n",
+    "    Args:\n",
+    "        OriginImage (np.ndarray): Any size of color image.\n",
+    "        InputImage (np.ndarray): The input of the model before dimension transposed and expanded.\n",
```

Comment:
Please use RST docstring format instead of Google docstrings. Example of how to do it presented in [notebook 403](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb), also please update parameters names.